.github/workflows/cypress.yml

```diff
@@ -0,0 +1,16 @@
+name: cypress-test-run
+on: [push]
+jobs:
+
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
It seems that v3 is available (however, not sure what is the difference): https://github.com/actions/checkout